README.md

```diff
@@ -0,0 +1,270 @@
+
+<!--
+HOW TO USE:
+This is an example of how you may give instructions on setting up your project locally.
+
+Modify this file to match your project and remove sections that don't apply.
+
+REQUIRED SECTIONS:
+- Table of Contents
+- About the Project
+  - Built With
+  - Live Demo
+- Getting Started
+- Authors
+- Future Features
+- Contributing
+- Show your support
+- Acknowledgements
+- License
+
+OPTIONAL SECTIONS:
+- FAQ
+
+After you're finished please remove all the comments and instructions!
+-->
+
+<div align="center">
+  <!-- You are encouraged to replace this logo with your own! Otherwise you can also remove it. -->
+  <!-- <img src="murple_logo.png" alt="logo" width="140"  height="auto" />
+  <br/> -->
+
+  <!-- <h3><b>Microverse README Template</b></h3> -->
+
+</div>
+
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#triangular_flag_on_post-deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 Web Developers Conference <a name="about-project"></a>
+
+
+**Web Developers Conference** is a project created to showcase the skills and knowledge I gained in Microverse Module 1.
+Have a look at [this presentation](https://www.loom.com/share/cb3e7a960be04030a818d5faea4feafc) for more detail.
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+<!-- > This portfolio website is creatd using HTML, CSS, and JavaScript. -->
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="https://www.w3schools.com/html/">HTML</a></li>
+    <li><a href="https://www.w3schools.com/css/">CSS</a></li>
+    <li><a href="https://www.w3schools.com/javascript/">JavaScript</a></li>
+  </ul>
+</details>
+
+<details>
+  <summary>Server</summary>
+  <ul>
+    <li>N/A</li>
+  </ul>
+</details>
+
+<details>
+<summary>Database</summary>
+  <ul>
+    <li>N/A</li>
+  </ul>
+</details>
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+<!-- > The following are the key features of the application. -->
+
+- **The conference page has home and about page. The home page contains the speakers, past conferences, welcome, and partners sections**
+- **The about page has information about the logo.**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+<!-- > Click the live demo to access the application. -->
+
+- [Live Demo Link](https://zewdiemc.github.io/Portfolio/)
+
```

Comment:
- [ ] [REQUIRED] Good job so far! Please remove `Portfolio` ❌ live demo link to current `Capstone` ✔️  project's live demo link 🙏🏼

Reply:
Fixed, please [have a look](https://github.com/ZewdieMc/web-development-course-capstone/pull/1/commits/2f1c91cfdc0fb2f731b4607a53ba9ec6f06f577f)

Reply:
Fixex, please [please see](https://github.com/ZewdieMc/web-development-course-capstone/pull/1/commits/c7b0e8d2cadedc0177471f071a044a1dbaae5c58).